widget/diagramwidget/.gitignore

```diff
@@ -0,0 +1,21 @@
+# Mac OS X files
+.DS_Store
+
+# Binaries for programs and plugins
+*.exe
```

Comment:
these files won't be in the widget package. I suspect that this file could be  removed - as we keep a .gitignore at the root.